rpi3.mk

```diff
@@ -44,9 +47,9 @@ ifeq ($(CFG_TEE_BENCHMARK),y)
 all: benchmark-app
 clean: benchmark-app-clean
 endif
-all: arm-tf optee-os optee-client xtest u-boot u-boot-rpi-bin\
+all: arm-tf buildroot optee-os optee-client xtest u-boot u-boot-rpi-bin\
```

Comment:
builtroot is managing `optee-client`, `optee-examples` and `xtest`, no need for that here or below in the clean

Reply:
I'll remove, what about the targets further down in the same file?
```Makefile
optee-client: optee-client-common 
... and friends
```
I can see that they are still in qemu.mk etc.

Reply:
Those should be removed too.
They where left by mistake in qemu.mk